configure.ac

```diff
@@ -101,8 +106,8 @@ dnl Check for gapi
 AC_PATH_PROG(GAPI_PARSER, gapi2-parser, no)
 if test "x$GAPI_PARSER" = "xno"; then
 	AC_MSG_CHECKING(for gapi2-parser.exe)
-	GAPI_PARSER=`which gapi2-parser.exe 2> /dev/null`
```

Comment:
I wonder what is this for, why would someone have the .exe in the PATH and why is it relying on which to find it instead of AC_PATH_PROG o_O